terraform/modules/cloud_sql/variables.tf

```diff
@@ -61,9 +61,14 @@ variable "sql_db_user_name" {
   description = "The name of the user. Changing this forces a new resource to be created."
 }
 
-variable "sql_database_name" {
+variable "sql_hono_database_name" {
   type        = string
-  description = "The name of the database in the Cloud SQL instance. This does not include the project ID or instance name."
+  description = "The name of the hono database in the Cloud SQL instance. This does not include the project ID or instance name."
```

Comment:
I would suggest to change it to this (There are other places too - if we decide to change - we should change all places)
> ... the Google Project ID ...

Reply:
Done